### ISSUE TITLE
Prettify not working when “JSON body” contains variable

### ISSUE THREAD

**Issue by bogmart** [usebruno/bruno#1553]
Hello,
I noticed a bug related to “Prettify” functionality when I put variables in 
![Screenshot (251)_bruno_err](https://github.com/usebruno/bruno/assets/36724163/7a05ca37-0270-4285-a52a-7f6976996f39)
a “JSON body”.

Example:
I have a variable named " input_idx" (it is an integer).
The “Prettify” for this “JSON body” is not possible:
{"method": "test", "params": {"input" : {{input_idx}} }}

![Screenshot (251)_bruno_err](https://github.com/usebruno/bruno/assets/36724163/15f7a364-156b-4298-b13b-675ee145f8db)

**Comment by nlassaux** [comment]
Are you actually able to access the variable? I can't manage to get bruno to work at all if I pass a variable the way you did

**Comment by bogmart** [comment]
> Are you actually able to access the variable? I can't manage to get bruno to work at all if I pass a variable the way you did

Yes, the variable is correctly expanded/replaced by Bruno with its value (in my case, Bruno sends a  HTTP POST). Only the prettify functionality is affected.
I just retested with Bruno v1.11.0, on WSL Ubuntu 22.04.

**Comment by nlassaux** [comment]
Got it, thanks. Super weird, in my case Bruno sends the variable with the curly brackets to the backend. I'm closely monitoring #1812 in case that fixes it at the same occasion

**Comment by bogmart** [comment]
Update: on version v1.17.0, the prettify generates an invalid JSON from my example.

```
{
  "method": "test",
  "params": {
    "input": {
      {input_idx
      }
    }
  }
}
```

**Comment by nathanmcgarvey-modopayments** [comment]
Also see https://github.com/usebruno/bruno/issues/3019

**Comment by helloanoop** [comment]
PR - #5113 has been merged that fixes this issue. This will be available in our upcoming - `v2.14.0` release scheduled to go out in last week of October
